Rules/CommonScripts/accolade_data.cfg

```
@@ -1661,6 +1751,12 @@ frodo123CZ =
 FrostGalaxyGamer =
 	participation 1;
 	# participation - US 3v3 TDM tournament no. 1
+	
+FuzzyBlueBaron =
+    participation 1;
```

```suggestion
	participation 1;
```

Fix indentation here.